user-guide/Cloud_Platform/CloudAdminApp/Auditing.md

```diff
@@ -9,3 +9,30 @@ If you have the Owner or Admin role for an organization on dataminer.services, y
 This page shows an overview of all the security-related events on dataminer.services that have happened for all systems connected to dataminer.services within your organization.
 
 To quickly find something in the potentially vast number of logs, use the filters at the top of the page.
+
+## Exporting the audit records
+
+<!-- RN 37164 -->
+
+To export the audit records to CSV:
+
+1. Click the *Export* button at the top of the page.
+
+1. In the pop-up window:
+
+   - Select whether a comma or semicolon should be used as separator.
+
+   - If column titles should be included, enable the option *Include column titles*.
+
+   - Click *Submit*.
+
+1. When you see a notification in the lower right corner mentioning that the audit record request has succeeded, check your email.
```

Comment:
The notification in the UI tells the 'request' is 'accepted' and we 'will send' an email. It goes fast now, but it could take a while when a lot of people are requesting a lot of big audit record exports. It's handled in the background.